### PR TITLE
Klanens stregliste 2.0

### PR DESCRIPTION
**Pull request by LarsRefsgaard**
This marks release 2.0 with:

1. A better header which stays sticky when scrolling and minimizes as well.
2. A better UI with tailwind, which allowed us to make a better dark mode with more consistent coloring.
3. Moved to Vite for better HMR and tooling in general.
4. Using Cypress to test basic functions in the app and allowing us to further test stuff like a navbar and logging in, in the future.
5. Using Pinia to have a shared state in a store so all components can for example know if the navbar is currently adding a person.
6. Also added firebase 9 sdk for tree-shakeable imports which reduces load times.

### CODE REVIEW

**Comment by lkbr1808** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/lkbr1808/klan-stregliste) found an issue: Missing radix parameter

It's currently on:
[cypress\integration\ui.buttons.spec.ts:17](https://github.com/lkbr1808/klan-stregliste/blob/4f5e446cef7289456e9a505d6b4ec1284884abed/cypress/integration/ui.buttons.spec.ts#L17)
Commit 4f5e446cef7289456e9a505d6b4ec1284884abed

**Comment by lkbr1808** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/lkbr1808/klan-stregliste) found an issue: Missing radix parameter

It's currently on:
[cypress\integration\ui.buttons.spec.ts:29](https://github.com/lkbr1808/klan-stregliste/blob/4f5e446cef7289456e9a505d6b4ec1284884abed/cypress/integration/ui.buttons.spec.ts#L29)
Commit 4f5e446cef7289456e9a505d6b4ec1284884abed

**Comment by lkbr1808** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/lkbr1808/klan-stregliste) found an issue: Missing radix parameter

It's currently on:
[cypress\integration\ui.buttons.spec.ts:51](https://github.com/lkbr1808/klan-stregliste/blob/4f5e446cef7289456e9a505d6b4ec1284884abed/cypress/integration/ui.buttons.spec.ts#L51)
Commit 4f5e446cef7289456e9a505d6b4ec1284884abed

**Comment by lkbr1808** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/lkbr1808/klan-stregliste) found an issue: Missing radix parameter

It's currently on:
[cypress\integration\ui.buttons.spec.ts:46](https://github.com/lkbr1808/klan-stregliste/blob/4f5e446cef7289456e9a505d6b4ec1284884abed/cypress/integration/ui.buttons.spec.ts#L46)
Commit 4f5e446cef7289456e9a505d6b4ec1284884abed

**Comment by lkbr1808** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/lkbr1808/klan-stregliste) found an issue: Missing radix parameter

It's currently on:
[cypress\integration\ui.buttons.spec.ts:22](https://github.com/lkbr1808/klan-stregliste/blob/4f5e446cef7289456e9a505d6b4ec1284884abed/cypress/integration/ui.buttons.spec.ts#L22)
Commit 4f5e446cef7289456e9a505d6b4ec1284884abed

**Comment by lkbr1808** [comment]
[CodeFactor](https://www.codefactor.io/repository/github/lkbr1808/klan-stregliste) found an issue: Missing radix parameter

It's currently on:
[cypress\integration\ui.buttons.spec.ts:22](https://github.com/lkbr1808/klan-stregliste/blob/4f5e446cef7289456e9a505d6b4ec1284884abed/cypress/integration/ui.buttons.spec.ts#L22)
Commit 4f5e446cef7289456e9a505d6b4ec1284884abed

File: src/components/UserList.vue [hunk 1]
```diff
@@ -1,52 +1,31 @@
...
-    db.collection('stregliste-mvp')
-      .orderBy('display-name', 'asc')
-      .onSnapshot((res) => {
-        stregliste.value = res;
-      });
+const stregliste = ref<Record<string, any>>({});
 
-    return { stregliste };
-  },
+onSnapshot(query(db, orderBy('display-name', 'asc')), (querySnapshot) => {
```
Comment (LarsRefsgaard) [inline src/components/UserList.vue:27]: asd

**Pull request merged**